packages/integration-react/src/components/LoadingPage/LoadingPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+import './LoadingPage.css';
+
+export class LoadingPage extends React.Component {
```

Comment:
Only thing is you might want to pass props down and spread them too the div around the bullseye ... rest looks good.